packages/components/tree-v2/src/hooks/useCheck.ts

```diff
@@ -0,0 +1,220 @@
+import { nextTick, ref, watch } from 'vue'
+import { NODE_CHECK_CHANGE, NODE_CHECK } from '../emits'
+import type { Ref } from 'vue'
+import type {
+  ITreeProps,
```

Comment:
In order to unify the naming, I think the `I` prefix is redundant, and the first letter of the typescript type can be capitalized.

Reply:
BTW, the props name should be `componentNameProps`(e.g `affixProps`), and the type of props should be `ComponentNameProps`(AffixProps). And so on.